code/modules/mob/language/station.dm

```diff
@@ -125,19 +125,6 @@
 	flags = WHITELISTED
 	syllables = list("qr","qrr","xuq","qil","quum","xuqm","vol","xrim","zaoo","qu-uu","qix","qoo","zix")
 
-/datum/language/skrellfar
-	name = LANGUAGE_SKRELLIANFAR
```

Comment:
If far Skrellian is being removed; might wanna look into code. I think SOMEWHERE was implant specifically to allow non-skrell to speak it that might need cleaning up.

Reply:
Yeah there was a few places that added LANGUAGE_SKRELLIANFAR and I went ahead and removed them. The way I ensured it was that if LANGUAGE_SKRELLIANFAR was mentioned anywhere in the code after I removed its define it would error out (Which is what happened with the EVENT1 species).